### ISSUE TITLE
transfer weight from kaldi to pytorch

### ISSUE THREAD

**Issue by JazminVidal** [cvqluu/Factorized-TDNN#4]
Hi!

How are you? 
I was wondering if there is a way to load thes weight of a pretrained kaldi network into this architechture you are implementing. Say, going from http://kaldi-asr.org/models/m13 to your architecture. Did you encounter this problem before and per chance solved it?

Thanks anyways :D

**Comment by cvqluu** [comment]
Hi, this is something I had a look at, and got as far as loading the weights into the model, but as far as I can tell there's a fundamental difference in the way the padding/edges of the convolutions are handled in kaldi from what I could implement in pytorch, meaning I always got a slightly mismatched output from kaldi.

I did only look at this for a few hours, and almost a year ago now, so I definitely didn't explore all options or fully work out what exactly was mismatched.

If you do manage to figure this out I'd be interested to see what you did :)
Chau
